installer/apt/install-debian-php-sury.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+apt-get -y install \
+  apt-transport-https \
+  ca-certificates \
+  curl
+
+wget -O "/etc/apt/trusted.gpg.d/php.gpg" "https://packages.sury.org/php/apt.gpg"
```

Comment:
This should go in /usr/share/keyrings and the apt sources line follow https://wiki.debian.org/DebianRepository/UseThirdParty

Reply:
Oh nice to know these recommendation !

Just updated the changes.